src/fklearn/training/classification.py

```diff
@@ -230,6 +231,20 @@ def p(new_df: pd.DataFrame, apply_shap: bool = False) -> pd.DataFrame:
 xgb_classification_learner.__doc__ += learner_return_docstring("XGboost Classifier")
 
 
+@curry
+def get_catboost_shap_values(df: pd.DataFrame, cbr: CatBoostClassifier,
+                             features: List, target: str,
+                             weights: List, cat_features: List) -> np.array:
+    """
+    """
+    import catboost
+    dtrain = catboost.Pool(df[features].values, df[target].values, weight=weights,
+                           feature_names=list(map(str, features)),
+                           cat_features=cat_features)
+    return cbr.get_feature_importance(type=catboost.EFstrType.ShapValues,
+                                      data=dtrain)
+
+
```

Comment:
afaik you can use shap on catboost directly from TreeExplainer, exactly as you do on lgb (L548) isn't it better to standardize?

Reply:
Nice! I was looking for it. This will enable us remove this auxiliar method 🚀

Reply:
I've updated the description with the reason to keep this auxiliar method. It's not the best, but does the job